test/resources/ns_with_lots_of_meta.clj

```diff
@@ -1,12 +1,18 @@
-(ns ^:md1 ^:md2 ^{:longhand "as well" :really? "yes" :ok true} ns-with-gen-class-methods-meta
+(ns ^:md1 ^:md2 ^{:longhand "as well" :really? "yes" :ok true}
+ ns-with-gen-class-methods-meta
   (:gen-class
    :methods [^:static [foo [String] String]
              ^{:test true} [bar [String] String]
              ^{:other "text"} [baz [String] String]]
    :name Name)
-  (:require [clojure.string :as s]
```

Comment:
I see `clojure.string` was already present here as of `master`.

This means this PR changes the intent of `[clojure.string]` from "should be cleaned up" to "should not be cleaned up".

This results in a loss of test coverage; I'd suggest adding an extra libspec so that both things will be exercised

Reply:
Yeah, but that's fine.  We have tons of other tests that tests removal.  I added another ns form that gets removed 🤷